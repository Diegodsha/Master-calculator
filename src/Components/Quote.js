import React from 'react';
import quoteImg from '../assets/quote.jpg';

const Quote = () => (
  <div style={{ maxWidth: '537px', margin: 'auto' }} className="card">
    <img src={quoteImg} className="card-img-top" alt="quote" />
    <div className="card-body">
      <blockquote className="blockquote mb-0">
        <p>
          Without mathematics, thereâ€™s nothing you can do. Everything around you
          is mathematics. Everything around you is numbers.
        </p>
        <footer className="blockquote-footer">
          Shakuntala Devi
          <cite title="Source Title"> - Indian writer and mental calculator</cite>
        </footer>
      </blockquote>
    </div>
  </div>
);

export default Quote;
