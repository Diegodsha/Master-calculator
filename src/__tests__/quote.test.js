import renderer from 'react-test-renderer';
import Quote from '../Components/Quote';

describe('Quote component', () => {
  it('Renders correctly Quote component', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
