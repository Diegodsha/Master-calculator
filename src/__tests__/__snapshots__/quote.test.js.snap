// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component Renders correctly Quote component 1`] = `
<div
  className="card"
  style={
    Object {
      "margin": "auto",
      "maxWidth": "537px",
    }
  }
>
  <img
    alt="quote"
    className="card-img-top"
    src="quote.jpg"
  />
  <div
    className="card-body"
  >
    <blockquote
      className="blockquote mb-0"
    >
      <p>
        Without mathematics, thereâ€™s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
      </p>
      <footer
        className="blockquote-footer"
      >
        Shakuntala Devi
        <cite
          title="Source Title"
        >
           - Indian writer and mental calculator
        </cite>
      </footer>
    </blockquote>
  </div>
</div>
`;
